awx_collection/plugins/modules/tower_job_launch.py

```diff
@@ -127,8 +127,8 @@ def update_fields(module, p):
     try:
         ask_extra_vars = tower_cli.get_resource('job_template').get(name=job_template)['ask_variables_on_launch']
         survey_enabled = tower_cli.get_resource('job_template').get(name=job_template)['survey_enabled']
```

Comment:
Oh I should have commented about this before but it slipped by me. This will cause 2 requests. It will be better to make 1 request, save `tower_cli.get_resource('job_template').get(name=job_template)` to a variable, and lookup survey_enabled inside of that.